 # Details of a polygonArray.
Header header
geometry_msgs/Polygon[] polygons
